src/app/comments/comment-form-container/comment-form/comment-form.component.css

```diff
@@ -0,0 +1,156 @@
+.comment-form{
+  margin: auto 0;
+  display: grid;
+  grid-template-columns: [line1] 30% [line2] 30% [line3] 40%;
+  grid-template-rows:  repeat(2, 1fr) 0.2fr repeat(2, 1fr) 0.2fr 1fr 0.2fr 2fr;
+  column-gap: 5px;
+}
+
+input, textarea {
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+#firstName_label {
+  grid-column: 1;
+  grid-row: 1 ;
+  align-self: end;
+}
+#firstName {
+  grid-column: 1;
+  grid-row: 2;
+}
+
+#lastName_label {
+  grid-column: 2;
+  grid-row: 1 ;
+  align-self: end;
+}
+#lastName {
+  grid-column: 2;
+  grid-row: 2;
+}
+
+
+#email_label {
+  grid-column: 3;
+  grid-row: 1 ;
+  align-self: end;
+}
+#email {
+  grid-column: 3;
+  grid-row: 2;
+}
+
+#body_label {
+  grid-column: 1;
+  grid-row: 4;
+  align-self: end;
+}
+#body {
```

Comment:
suggest add comment- prefix to name, `#body ` is too similar to html body element